tap_redshift/__init__.py

```diff
@@ -193,8 +192,21 @@ def schema_for_column(c):
 def create_column_metadata(cols):
     mdata = metadata.new()
     mdata = metadata.write(mdata, (), 'selected-by-default', False)
+
     for c in cols:
+        valid_rep_keys = [c['name'] for c in cols
+                          if c['type'] == 'timestamp'
+                          or c['type'] == 'timestamptz']
```

Comment:
There are more timestamp types that should be considered here (i.e. all in `DATETIME_TYPES` except date). Perhaps you should split `DATETIME_TYPES` as `DATE_TYPES` and `DATETIME_TYPES` and re-use the latter here as opposed to hardcoding types?